File: LeetCode268.java
class Solution {
    public int missingNumber(int[] nums) {
        int length = nums.length;
        int sum = 0;
        int sum1 = 0;
        for (int i = 0; i < length + 1; i++) {
            sum = sum + i;
        }
        for (int i = 0; i < length; i++) {
            sum1 = sum1 + nums[i];
        }
        return sum - sum1;
    }
}

File: LeetCode268.py
class Solution(object):
    def missingNumber(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        l = len(nums)
        ll = l + 1
        suml = 0
        sumll = 0
        for x in nums:
            suml = suml + x
        for y in range(ll):
            sumll = sumll + y
        res = sumll - suml
        return res